### PR TITLE
Improved translations for NL and NL_BE

### PR DESCRIPTION
**Pull request by DjedenOfficial**
I improved the translations for NL and NL_BE. Now it has been fully translated instead of partly. The changes haven't been tested. I would like to know how to test since I don't know how (I am new to GitHub).

### CODE REVIEW

**Comment by ia** [comment]
> I would like to know how to test since I don't know how

Hello. Once this workflow will be approved to run tests & to prepare builds[^1], and if everything is ok, then you will see pre-built binaries as downloadable artifacts here which you can download, flash & test on hardware which you have.

[^1]: Usually it happens automatically once any pull-request is made but it seems something has been changed recently on that so now it seems like admins of the project have to trigger it manually.

**Comment by Ralim** [comment]
Each new contributor needs to be approved once before the automated actions can run.

So now they should always work for you @DjedenOfficial.

Once they finish you should be able to download the compiled code from them for testing

**Comment by ia** [comment]
I'm not an expert on git/github myself, but maybe another reason it's because you did make a commit not into a separate branch with different name (after you forked the repo) but into `dev` branch (which is "protected" branch because it's "_upstream_" one).

Bottom line is just a suggestion for the future, here is the recommended way to make pull-requests, @DjedenOfficial:
- fork the repo to your account (i.e., using github interface)[^1];
- clone forked repo locally: `git clone ...`[^1];
- create your own branch for your own changes (it seems this step has been skipped - that's why actions maybe haven't been triggered automatically): `git checkout -b mychange dev`, where `mychange` - name of your branch, `dev` - name of upstream branch of forked repo;
- add changes / fixes by editing files;
- make a commit: `git commit ...`;
- push your local branch with your local changes to your forked repo on github: `git push -u YOUR_GITHUB_USERNAME mychange`;
- after that once you open github it will show you that you did make a push into forked repo and github will suggest you to create pull-request automatically.

[^1]: these steps should be done only once per each repo/project

Something like that.

**Comment by ia** [comment]
> I would like to know how to test

Here you go, [now here you can download](https://github.com/Ralim/IronOS/actions/runs/6605573287#artifacts) binaries with your changes to test.

**Comment by DjedenOfficial** [comment]
> > I would like to know how to test
> 
> Here you go, [now here you can download](https://github.com/Ralim/IronOS/actions/runs/6605573287#artifacts) binaries with your changes to test.

Thanks testing went well I fixed some minor issues and want to run another test.

**Comment by Ralim** [comment]
All good, Ill approve until this merges; so don't sweat and just give me a bit to see the email 😁

**Comment by Ralim** [comment]
@DjedenOfficial Are you happy with how this looks now ?

**Comment by DjedenOfficial** [comment]
> @DjedenOfficial Are you happy with how this looks now ?

Sorry for not reacting. It needs just slight changes but I will be making them as soon as possible and are likely to be done by the afternoon. Is there a way to test firmware by not making a whole pull request?

**Comment by Ralim** [comment]
No rush, just that it went silent so I wasn't sure.

You can compile locally to test; or you can setup your fork to run the builds for testing.

Convention is to mark a PR as a draft until you're good to go (but don't stress, everyone misses the option)

File: Translations/translation_NL.json
Comment (ColoMAX) [inline Translations/translation_NL.json:7]: Calibratie\nklaar!

File: Translations/translation_NL.json
Comment (ColoMAX) [inline Translations/translation_NL.json:13]: Instellingen\nzijn veranderd

File: Translations/translation_NL.json
Comment (ColoMAX) [inline Translations/translation_NL.json:16]: old was correct

File: Translations/translation_NL.json
Comment (ColoMAX) [inline Translations/translation_NL.json:22]: Old was correct

File: Translations/translation_NL.json
Comment (ColoMAX) [inline Translations/translation_NL.json:25]: Old was correct

File: Translations/translation_NL.json
Comment (ColoMAX) [inline Translations/translation_NL.json:28]: Old was correct

File: Translations/translation_NL.json
Comment (ColoMAX) [inline Translations/translation_NL.json:31]: Thermische catastrofe?

File: Translations/translation_NL.json
Comment (ColoMAX) [inline Translations/translation_NL.json:40]: lower case and "aan het calibreren" if it fits on screen.

File: Translations/translation_NL.json
Comment (ColoMAX) [inline Translations/translation_NL.json:52]: Too long: "Ingangs V: \n"

File: Translations/translation_NL.json
Comment (ColoMAX) [inline Translations/translation_NL.json:73]: +"...een namaak!"

File: Translations/translation_NL.json
Comment (ColoMAX) [inline Translations/translation_NL.json:109]: Old was more Dutch sounding

File: Translations/translation_NL.json
Comment (ColoMAX) [inline Translations/translation_NL.json:120]: Undo this

File: Translations/translation_NL.json
Comment (ColoMAX) [inline Translations/translation_NL.json:128]: QC is a name, you can not translate this away

**Review by ColoMAX — CHANGES_REQUESTED** [review]
From a quick glance, I've found at least these issues.

**Comment by discip** [comment]
@DjedenOfficial & @ColoMAX
It would be very nice if you could find a solution that you both could agree on. 😊

**Comment by discip** [comment]
@DjedenOfficial & @ColoMAX
It would be very nice if you could find a solution that you both could agree on, as neither the developer nor anyone else authorized to merge this in are knowledgeable enough to judge what is best. 😊

thanks

**Comment by discip** [comment]
@DjedenOfficial & @ColoMAX

Anyone?

**Comment by discip** [comment]
@DjedenOfficial & @ColoMAX

Any progress yet?

**Comment by discip** [comment]
@DjedenOfficial & @ColoMAX

We need a solution guys. Please answer.

**Comment by discip** [comment]
@ColoMAX
> Later this week I can try to add a commit to the pr to resolve all

This was 3 weeks ago, are you still interested, if not I will merge this in as is.

**Comment by discip** [comment]
@ColoMAX
> Later this week I can try to add a commit to the pr to resolve all

This was 3 weeks ago.
Are you still interested?
If not I'll merge this in as is.

**Comment by ColoMAX** [comment]
@DjedenOfficial can you add me as a contributor on your fork? I can not push my changes to your repo.
Or is there another way to do it?
I've worked on the Dutch translations only, not the Flemish though, since I'm not familiar with it. Although one might consider, for the time being, to keep it the same as the Dutch until somebody comes around who does, due to the similarities between the two languages/dialects.

**Pull request closed without merging**
